templates/artist/artist_detail.html

```diff
@@ -104,9 +104,9 @@
             var myChart5 = new Chart(ctx, {
                 type: 'pie',
                 data: {
-                    labels: ["Me", "Other Investors", "Artist"],
+                    labels: ["Me", "Artist", "Other Investors", "Brands"],
```

Comment:
Changing the order of labels, but not reordering the data below will (I think) result in mislabelling.